package.json

```diff
@@ -28,12 +28,11 @@
     "cypress": "14.5.0"
   },
   "dependencies": {
-    "@metamask/detect-provider": "2.0.0",
     "@nuxtjs/robots": "5.2.11",
     "@nuxtjs/tailwindcss": "6.14.0",
     "@pinia/nuxt": "0.11.1",
-    "@reown/appkit": "1.7.10",
-    "@reown/appkit-adapter-ethers": "1.7.10",
+    "@reown/appkit": "1.7.11",
```

Comment:
For these can you update them in a PR against `main` and we can rebase this PR on top of it?